CHANGES.rst

```diff
@@ -18,6 +18,7 @@ Unreleased
     decorators. :issue:`4104`
 -   Fixed the issue where typing requires template global
     decorators to accept functions with no arguments. :issue:`4098`
+-   Update docs of register view, to return explanation if insert to db failed. :issue:`4099`
```

Comment:
I don't think updates to the example need a change log entry. It's not useful information for people who want to see what changed in an update.

Reply:
It is removed now. Thanks